### ISSUE TITLE
Option to preserve blank lines

### ISSUE THREAD

**Issue by gempain** [Leonidas-from-XIV/node-xml2js#420]
Hello,

Is there a way to preserve blank lines when doing an xml -> js -> xml transform ? Suppose I parse:

```
<test>
    <a></a>

    <b></b>
</test>
```

Make some modifications and convert it back to xml, I'd expect something like:

```
<test>
    <a>World</a>

    <b></b>
</test>
```

but instead I get

```
<test>
    <a>World</a>
    <b></b>
</test>
```

Thanks for the great plugin :)

All the best;

**Comment by Leonidas-from-XIV** [comment]
Not really. It is a bit of an issue, since technically there should be a text node between the `<a>` and `<b>` nodes but the parsing semantics of xml2js can be a bit — surprising to say in the least.

I wish I could revisit the whole logic some day.

**Comment by gempain** [comment]
No worries, I guess this can be a feature request then :) Thanks for the answer :)
